README.md

```diff
@@ -31,7 +31,7 @@ require("expose-loader?libraryName!./file.js");
 
 For example, let's say you want to expose jQuery as a global called `$`:
 
-```
+```javascript
```

Comment:
Could you please use `js` instead ? :)

Reply:
Ahhh I've always preferred the full language names for clarity. But sure :)